docker-hub/builds/index.md

```diff
@@ -299,6 +303,10 @@ You can enable the BuildKit builder by setting the `DOCKER_BUILDKIT=1`
 Refer to the [build images with BuildKit](../../develop/develop-images/build_enhancements.md)
 page for more information on BuildKit.
 
+>**Note**
+>
+>Autobuilds use the default build-system. If you want to use the old Docker build-system, use DOCKER_BUILDKIT=0.
```

Comment:
I think we need to be more explicit here and say something like: "Autobuilds use the buildKit build-system by default. If you want to use the legacy Docker build-system, add the environment variable `Docker_BUILDKIT=0` (maybe with a link to adding environment variables on Autobuilds higher in the page)